Issue 4903 - Empty rel and noFollow

Marko64 opened the pull request:
Fixed link rel attributes for link and text-link components.

Fixes #4903  


**_ Front/Back Testing _**

-   [ ] Insert a text and add a link to it. Make sure there is no `rel` attribute added to the `a` tag in HTML. Try adding `nofollow`, `sponsored` and `ugc` and make sure the `rel` tag shows normally now. 
-   [ ] Test the same with a different block, such as an image.

**_ Pre-Code Testing _**

-   [ ] Insert a text and add a link to it. Make sure there is no `rel` attribute added to the `a` tag in HTML. Try adding `nofollow`, `sponsored` and `ugc` and make sure the `rel` tag shows normally now. 
-   [ ] Test the same with a different block, such as an image.
-   [ ] Check no commented code and no unnecessary imports
-   [ ] Standards of the project have been followed
-   [ ] No errors/warnings on console

# Checklist

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] My changes generate no new warnings/errors
-   [ ] I have added/updated tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes

Marko64 committed:
Issue 4903 - Empty rel and noFollow

Olekrut reviewed (changes requested):
![image](https://github.com/maxi-blocks/maxi-blocks/assets/88893350/833097d5-8161-4197-a531-dfbf5d87f006)
the same for master and this brunch

Marko64 committed:
Update fonts.js

I think let's remove it then since it only works with internal pages and the text can't be changed. If we need the title, we can do it better in the futur.

Olekrut reviewed (changes requested):
![image](https://github.com/maxi-blocks/maxi-blocks/assets/88893350/0d7d8e15-3f59-4bb2-94fa-d57b9fb7d87a)
title for UGC

Marko64 committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into issue-4903-link-rel-empty-nofollow-not-working

Marko64 committed:
snapshots

Marko64 committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into issue-4903-link-rel-empty-nofollow-not-working

Marko64 committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into issue-4903-link-rel-empty-nofollow-not-working

Marko64 committed:
Update index.spec.js.snap

Marko64 committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into issue-4903-link-rel-empty-nofollow-not-working

Marko64 committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks into issue-4903-link-rel-empty-nofollow-not-working

Marko64 committed:
tests

Olekrut reviewed (changes requested):
1) ![image](https://github.com/maxi-blocks/maxi-blocks/assets/88893350/787013ff-e5e8-4050-a6db-16068e11375b)
is it ok?
if set type just #, we will have empty title..

2) is not correct link
![image](https://github.com/maxi-blocks/maxi-blocks/assets/88893350/b6eb1ef2-924c-447c-aecc-40c16221847e)

3) check pls
[links.txt](https://github.com/maxi-blocks/maxi-blocks/files/12260817/links.txt)
